package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "did-context",
-  "version": "2.0.1-0",
+  "version": "3.0.0",
```

Comment:
this line needs to be backed out, the publish script handles this.

Reply:
Fixed.